src/main/scala/org/broadinstitute/dsde/workbench/leonardo/db/ClusterComponent.scala

```diff
@@ -148,34 +148,24 @@ trait ClusterComponent extends LeoComponent {
 
     // find* and get* methods do query the INSTANCE table
 
-    def findByName(project: GoogleProject, name: ClusterName) = {
-      clusterQueryWithInstancesAndErrorsAndLabels.filter { _._1.googleProject === project.value }.filter { _._1.clusterName === name.value }
-    }
-
-    def getClusterByName(project: GoogleProject, name: ClusterName): DBIO[Option[Cluster]] = {
```

Comment:
This method is dangerous! Luckily it was only being used in tests. (project, name) is not a unique key on CLUSTER, so this was taking the `headOption` of potentially multiple records. `getActiveClusterByName` should be used instead, since it should return a unique cluster.